.buildkite/pipeline.yml

```diff
@@ -63,20 +63,22 @@ steps:
   - label: contract doc development
     if: build.branch != "autodoc/master" && build.branch != "master"
     commands:
-      - nix-build ci.nix -A contracts-doc --arg release false
-      - ln -s ./result/BaseDAO.md BaseDAO.md
+    - nix-build ci.nix -A contracts-doc --arg release false
+    - ln -s ./result/*.md .
     artifact_paths:
       - BaseDAO.md
+      - GameDAO.md
 
   - label: contract doc release
     if: build.branch == "master"
     commands:
     - nix-build ci.nix -A contracts-doc --arg release true
         --argstr commitSha "$(git rev-parse HEAD)"
         --argstr commitDate "$(git log HEAD -1 --format=%cd)"
-    - ln -s ./result/BaseDAO.md BaseDAO.md
+    - ln -s ./result/*.md .
     artifact_paths:
       - BaseDAO.md
+      - GameDAO.md
```

Comment:
Why people move files to `.` and then refer to specific files, instead of referring to the folder :unamused:

If not this, we could decrease the number of necessary updates on contract addition to 1-2.

Reply:
Maybe because this way it's easier to download these artifacts from CI :shrug:

Reply:
Fair, probably that's the reason